test/benchmark/lmbench-exec/run.sh

```diff
@@ -9,5 +9,6 @@ echo "*** Running the LMbench exec latency test ***"
 if [ ! -d /tmp ]; then
     mkdir /tmp
 fi
+/sbin/ldconfig
```

Comment:
This should benefit all benchmark commands. So I think it should be added to the global entrypoint script.

```bash
generate_entrypoint_script() {
    local benchmark="$1"
    local init_script=$(cat <<EOF
#!/bin/sh
/sbin/ldconfig
mount -t devtmpfs devtmpfs /dev
modprobe virtio_blk
mkfs.ext2 -F /dev/vda
mount -t ext2 /dev/vda /ext2
```

Reply:
This `generate_entrypoint_script` is only worked for Linux. If we want to add `/sbin/ldconfig`, one way is adding this command to the `init_args` of OSDK. Yet the `init_args` currently cannot successfully parse consecutive commands (such as commands separated by && or ;) due to some bugs in the parsing mechanism.

Reply:
> This generate_entrypoint_script is only worked for Linux. 

You are right on this point. But it does not change the fact that adding duplicated pre-test steps is an unsustainable approach.

```
if [ ! -d /tmp ]; then
    mkdir /tmp
fi
/sbin/ldconfig
```

These lines have been repeated in multiple `run.sh`. We may forget to add these lines manually when new benchmark tests are added in the future. So I think they should be extracted into a centralized place.

For example, we can add a `runner.sh` under `benchmark/common`, which runs every benchmark.

```bash
#!/bin/sh

set -e

print_help() {
    echo "Usage: $0 bench_name"
    echo ""
    echo "The bench_name argument must be one of the directory under `<asterinas>/test/benchmark/`".
}

BENCH_NAME=$1
SCRIPT_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )

# Validate arguments
# ...

# Prepare the environment
if [ ! -d /tmp ]; then
    mkdir /tmp
fi
/sbin/ldconfig

# Run the benchmark
${SCRIPT_DIR}/$1/run.sh
```

The root `Makefile` can be modified to use `runner.sh` as the init program.